easyrest/views/restaurant_controler.py

```diff
@@ -0,0 +1,49 @@
+from pyramid.view import view_config
+from pyramid.response import Response
+
+from pyramid.httpexceptions import HTTPNotFound
+
+from sqlalchemy.exc import DBAPIError
+
+from json import dumps
+
+from ..models.Restaurant import Restaurant
+
+
+def asign_tags(rests):
+    rests_dict = [rest.as_dict() for rest in rests]
+    for i, rest in enumerate(rests):
+        del(rests_dict[i]["menu_id"])
+        rests_dict[i]["id"] = "restaurantId" + str(rests_dict[i]["id"])
+        tags = rest.tag
+        tags_list = [tag.as_dict() for tag in tags]
+        for tag in tags_list:
+            tag["id"] = "tagId" + str(tag["id"])
```

Comment:
Why do you update ids here?

Reply:
To use as unique key in React components.